CHANGES.rst

```diff
@@ -1,11 +1,14 @@
 Change Log
 ==========
 
-**Unreleased**
+Unreleased
+----------
 
-* :meth:`Submission.mod.set_original_content` support 
+**Added**
+
+* :meth:`.set_original_content` supports
   marking a submission as "original content."
```

Comment:
Can you please also remove the quotation marks? Thank you.

Reply:
Done! Sorry, forgot about that.